cli/gen_runtime_vectors/main.go

```diff
@@ -251,14 +251,27 @@ func main() {
 								{},
 								// Valid data, one function call with argument.
 								{
-									"say_hello": map[string]interface{}{
-										"who": "me",
+									"call": map[string]interface{}{
+										"say_hello": map[string]interface{}{
+											"who": "me",
+										},
 									},
 								},
 								// Valid data, one function call with argument.
 								{
 									"instantiate": map[string]interface{}{
-										"initial_counter": 42,
+										"instantiate": map[string]interface{}{
+											"initial_counter": 42,
+										},
+									},
+								},
+								// Valid data, multiple function calls.
```

Comment:
It's fine as a test vector (as any data is potentially valid since smart contracts can use any format they choose) but just note that this does not actually result in multiple calls under the current Contract SDK.